src/osd/ReplicatedBackend.cc

```diff
@@ -610,14 +608,11 @@ void ReplicatedBackend::submit_transaction(
       new C_OSD_OnOpApplied(this, &op)));
   op_t->register_on_applied(
     new ObjectStore::C_DeleteTransaction(op_t));
-  op_t->register_on_applied(
-    new ObjectStore::C_DeleteTransaction(local_t));
   op_t->register_on_commit(
     parent->bless_context(
       new C_OSD_OnOpCommit(this, &op)));
 
   list<ObjectStore::Transaction*> tls;
-  tls.push_back(local_t);
   tls.push_back(op_t);
```

Comment:
@athanatos , @liewegas , as  David Zafman said "I think this is where the create_collection() is being added to the transaction:"
But I do not find the "get_temp_coll " is master branch.